### PR TITLE
dual_clock fifo and bsvpreprocess.py

### PR DESCRIPTION
**Pull request by hanw**
Please take a look at first two commits.

### CODE REVIEW

File: scripts/bsvpreprocess.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def nexttok(s):
             else:
                 defs[sym] = ''
         elif tok == 'include':
-            m = re.search('"?([-_A-Za-z0-9.]+)"?', s)
+            m = re.search('"+([-_A-Za-z0-9.]+)"+', s)
```
Comment (jameyhicks) [inline scripts/bsvpreprocess.py]: Do you have examples where there are multiple starting or ending double quotes?

Comment (hanw) [inline scripts/bsvpreprocess.py]: I reverted this change, will submit again when I ran into an example of this issue.

**Comment by hanw** [comment]
Can I merge this?

**Comment by jameyhicks** [comment]
With the new UI for "review" I missed the merge step somehow.

**Comment by jameyhicks** [comment]
now that I've merged it, I should see if there is a way to merge in github now without a merge commit

**Pull request merged**